components/Modal/Modal.jsx

```diff
@@ -85,8 +88,10 @@ class Modal extends React.Component {
   }
 
   componentDidMount() {
-    const { body } = document;
-    body.appendChild(this.modalOverlay);
+    if (document.querySelector('#regression-tests')) {
```

Comment:
this condicional will be necessary in all components that use Portal (SnackBar for example).
How about of export in a method to abstract it.

Could be placed in `components/shared` folder, for example. 

called `regressionTestSelector` for example. WDYT?

Reply:
![](https://media1.giphy.com/media/QEYYlJqOaEhXrjTrOH/giphy.gif)